droidlet/dashboard/web/src/components/Timeline/Timeline.js

```diff
@@ -82,17 +89,23 @@ class DashboardTimeline extends React.Component {
     this.timeline = {};
     this.appRef = createRef();
     this.prevEvent = "";
+    this.searchPattern = "";
   }
 
   componentDidMount() {
     if (this.props.stateManager) this.props.stateManager.connect(this);
+
+    // make a shallow copy of search filters
+    this.searchFilters = [...this.props.stateManager.memory.timelineFilters];
+
     this.timeline = new Timeline(this.appRef.current, items, groups, options);
-    // set current viewing window to 10 seconds for readability
+    // set current viewing window to 20 seconds for readability
```

Comment:
isn't this 30 seconds(20 - (-10))?

Reply:
I've renamed the variables in [this commit](https://github.com/facebookresearch/droidlet/pull/631/commits/8a693810c31c907e18dd18670b8e4c8c7f58ae9d). To clarify the previous logic, since I was reusing using the same variable (currentTime) it's more like -10 seconds from the current time, then +20 seconds after that which results in +10. So -10 to 10 is a window of 20 seconds, if that makes sense?